<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsSurveyAttributesDialog</class>
 <widget class="QDialog" name="QgsSurveyAttributesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>444</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Survey Attributes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolBar" name="mToolbar">
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <addaction name="mActionToggleEditing"/>
     <addaction name="mActionSaveEdits"/>
     <addaction name="mActionReload"/>
     <addaction name="separator"/>
     <addaction name="mActionDeleteSelected"/>
     <addaction name="separator"/>
     <addaction name="mActionAggregateRounds"/>
     <addaction name="separator"/>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="mOptionsSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QFrame" name="mOptionsListFrame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="mOptionsListWidget">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Measurements</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Leveling</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GNSS base lines</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GNSS coordinates</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Points</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Stations</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Field Book</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="mOptionsFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="mOptionsStackedWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="mPageMeasurements">
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QTreeView" name="mMeasurementsView"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="mPageLeveling"/>
         <widget class="QWidget" name="mPageGNSSBaseLines"/>
         <widget class="QWidget" name="mPageGNSSCoordinates"/>
         <widget class="QWidget" name="mPagePoints">
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLineEdit" name="mPointsFilterEdit">
             <property name="placeholderText">
              <string>Search</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QTableView" name="mPointsView">
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="mPageStations">
          <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLineEdit" name="mStationsFilterEdit">
             <property name="placeholderText">
              <string>Search</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" rowspan="2">
            <widget class="QGroupBox" name="mStationsMetadataBox">
             <property name="title">
              <string>Metadata</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QListView" name="mStationsListView">
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="mPageFieldBook">
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QLineEdit" name="mFieldBookFilterEdit">
             <property name="placeholderText">
              <string>Search station or point ID</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QTreeView" name="mFieldBookListView"/>
           </item>
           <item row="0" column="1" rowspan="2">
            <widget class="QGroupBox" name="mFieldBookMetadataBox">
             <property name="title">
              <string>Metadata</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="mFieldBookMetadataLbl">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="mActionToggleEditing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionToggleEditing.svg</normaloff>:/images/themes/default/mActionToggleEditing.svg</iconset>
   </property>
   <property name="text">
    <string>Toggle Editing</string>
   </property>
   <property name="toolTip">
    <string>Toggle editing mode (Ctrl+E)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="mActionSaveEdits">
   <property name="icon">
    <iconset resource="../../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionSaveAllEdits.svg</normaloff>:/images/themes/default/mActionSaveAllEdits.svg</iconset>
   </property>
   <property name="text">
    <string>Save edits</string>
   </property>
   <property name="toolTip">
    <string>Save edits (Ctrl+S)</string>
   </property>
  </action>
  <action name="mActionReload">
   <property name="icon">
    <iconset resource="../../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionRefresh.svg</normaloff>:/images/themes/default/mActionRefresh.svg</iconset>
   </property>
   <property name="text">
    <string>Reload the table</string>
   </property>
   <property name="toolTip">
    <string>Reload the table</string>
   </property>
  </action>
  <action name="mActionDeleteSelected">
   <property name="icon">
    <iconset resource="../../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionDeleteSelectedFeatures.svg</normaloff>:/images/themes/default/mActionDeleteSelectedFeatures.svg</iconset>
   </property>
   <property name="text">
    <string>Delete selected features</string>
   </property>
   <property name="toolTip">
    <string>Delete selected features</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="mActionAggregateRounds">
   <property name="text">
    <string>Aggregate rounds</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>mOptionsListWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>mOptionsStackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
